addons/project/__manifest__.py

```diff
@@ -18,6 +18,7 @@
         'web',
         'web_tour',
         'digest',
+        'base_install_request',
```

Comment:
Why this dependence?
Shouldn't the `base_install_request` module be uninstallable if the feature is not needed?

Reply:
The module has to be *explicitely* defined in the dependencies to be used as

```py
'auto_install': ['base_install_request']
```

Sad but true.